File: generated-sources/scala-finch/mojang-api/src/main/scala/org/openapitools/models/Inline_object_1.scala
package org.openapitools.models

import io.circe._
import io.finch.circe._
import io.circe.generic.semiauto._
import io.circe.java8.time._
import org.openapitools._
import org.openapitools.models.SkinModel

/**
 * Request Mojang to download a skin from an URL and apply to the player
 * @param model 
 * @param url The URL which Mojang servers will download and apply the skin
 */
case class Inline_object_1(model: Option[SkinModel],
                url: String
                )

object Inline_object_1 {
    /**
     * Creates the codec for converting Inline_object_1 from and to JSON.
     */
    implicit val decoder: Decoder[Inline_object_1] = deriveDecoder
    implicit val encoder: ObjectEncoder[Inline_object_1] = deriveEncoder
}

File: generated-sources/javascript-closure-angular/mojang-api/API/Client/InlineObject1.js
goog.provide('API.Client.inline_object_1');

/**
 * Request Mojang to download a skin from an URL and apply to the player
 * @record
 */
API.Client.InlineObject1 = function() {}

/**
 * @type {!API.Client.SkinModel}
 * @export
 */
API.Client.InlineObject1.prototype.model;

/**
 * The URL which Mojang servers will download and apply the skin
 * @type {!string}
 * @export
 */
API.Client.InlineObject1.prototype.url;

